lib/ship.rb

```diff
@@ -0,0 +1,25 @@
+require 'pry'
```

Comment:
Do we have to require 'pry' on class files?

Reply:
No I just used it here, but it isnt necessary.